cubids/utils.py

```diff
@@ -1,10 +1,27 @@
 """Miscellaneous utility functions for CuBIDS."""
+
 import re
 from pathlib import Path
 
 
 def _get_container_type(image_name):
-    """Get and return the container type."""
+    """Get and return the container type.
```

Comment:
roadmap, we'll have to decide whether we want to keep container support. It's probably not worth it

Reply:
I've opened #285 about this.